views/js/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -423,6 +458,34 @@ define([
                 ];
                 var cke;
 
+                createTooltip($container, 'error', __('This is not a valid answer'), true, true);
+                var patternHandler = function patternHandler(e) {
+                    var isCke = _getFormat(interaction) === 'xhtml';
```

Comment:
@krampstudio I'm not sure if we still need this, and how to test it at all.

Reply:
I think it's required because when the interaction is using XHTML format, the textarea is replaced by CKEditor and the value can't get retrieved the same way.